src/app/components/shared/formly/image-input.component.ts

```diff
@@ -27,6 +27,12 @@ import { asFormControl } from "./helper";
           [formlyAttributes]="field"
           (ngModelChange)="readFile()"
         />
+
+        <button
+          type="button"
+          (click)="this.formControl.setValue('')"
```

Comment:
empty string is not the same as `null`. Use `null` - which means explicitly empty.

Also make this a function in the component rather than an inline callback.

Reply:
This has now been fixed